readme.md

```diff
@@ -186,82 +237,244 @@ const results = [
+- Update `<Search />` in the UI so that it takes in `this.state.query` via props as `query`
+- Update `<Results />` in the UI so that it takes in `this.state.shows` via props as `shows`
+
+## Catch-Up & Break (15 minutes / 2:05)
+
+> By this point, [your app should look something like this](https://github.com/ga-wdi-exercises/react-tvmaze/tree/474044ff43b809c001595c7b51842e82731d2fb0)
+
+## [Add Inverse Data Flow](https://facebook.github.io/react/docs/thinking-in-react.html#step-5-add-inverse-data-flow) (15 minutes / 2:20)
 
-### [Add Inverse Data Flow](https://facebook.github.io/react/docs/thinking-in-react.html#step-5-add-inverse-data-flow)
 The last step is passing callbacks through props to presentational components to provide behavior.
-We will need three functions defined on the `SearchContainer` component to provide necessary behavior to `Search` and `Results`:
+We will need three functions defined on the `SearchContainer` component to provide necessary behavior to `Search` and `Results`...
 
 1. `handleSearchInput` for managing changes to the input field
-2. `onSubmitQuery` for kicking of the ajax request
-3. `onSearchAgain` to set state back to render the search bar
+2. `onSubmitQuery` for switching `hasSearched` so that `Results` are displayed and making an AJAX call for searched TV shows
+3. `onSearchAgain` for switching `hasSearched` so that the search bar is displayed again
+
+### `onSubmitQuery`
+
+Let's first illustrate this concept by defining an `onSubmitQuery` method. For now, we will focus on switching `hasSearched` from `false` to `true`. We will save making the AJAX call for searched TV shows for later.
+
+#### In `SearchContainer.js`...
 
-> What is the point of all of these `.bind(this)` statements?
+Define an `onSubmitQuery` method in the component definition. It should use `.setState` to set `hasSearched` to `true`.
 
-We are also going to add a `Utils.js` file where we ultimately will make the request to TVMaze.
-For now we'll just hard code the mock data
+```js
+class SearchContainer extends Component {
+  // ...
+  onSubmitQuery(e) {
+    e.preventDefault()
+    this.setState({
+      hasSearched: true
+    })
+  }
+  // ...
+}
+```
+
+> Because the method will be trigger by a submit event, the `e` parameter is automatically populated with an event object.
+
+Because this method will be triggered when the user interacts with the `Search` component, we need to pass `onSubmitQuery` down to `Search` via props. We can do this with methods similarly to how we pass down data values...
+
+```js
+class SearchContainer extends Component {
+  // ...
+  render(){
+    const toRender = this.state.hasSearched
+      ? <Results shows={this.state.shows} />
+      : <Search query={this.state.query} onSubmitQuery={this.onSubmitQuery} />
+    return <div>{toRender}</div>
+  }
+}
+```
+
```

Comment:
I've used it in class a few times with them, but I'll show them different ways of implementing conditional logic here.